tests/cypress/fixtures/graphene.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Graphene",
+    "name": "Graphene (mp-1040425)",
```

Comment:
Can we take this from standata?

Reply:
I think I left it this way since we're testing Standata itself.